server/src/main/java/org/apache/druid/segment/indexing/DataSchema.java

```diff
@@ -83,6 +84,7 @@ public DataSchema(
       @JsonProperty("metricsSpec") AggregatorFactory[] aggregators,
       @JsonProperty("granularitySpec") GranularitySpec granularitySpec,
       @JsonProperty("transformSpec") TransformSpec transformSpec,
+      @JsonProperty("hasRolledUpSegments") @Nullable Boolean hasRolledUpSegments,
```

Comment:
`isRolledUp` would probably be a better name. What do you think?

Reply:
There could be a subset of segments which are rolled-up. So `isRolledUp` didn't sound right to me.